configure.py

```diff
@@ -1607,7 +1607,7 @@ def configure_seastar(build_dir, mode, mode_config):
         '-DCMAKE_C_COMPILER={}'.format(args.cc),
         '-DCMAKE_CXX_COMPILER={}'.format(args.cxx),
         '-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON',
-        '-DSeastar_CXX_FLAGS={}'.format((mode_config['lib_cflags']).replace(' ', ';')),
+        '-DSeastar_CXX_FLAGS=SHELL:{}'.format(mode_config['lib_cflags']),
```

Comment:
A challenging goal indeed, but I believe they managed to achieve it.

Reply:
I once read somewhere that they didn't go with some existing embeddable language because they didn't want CMake to be Turing complete. And now we have conditions, loops, variables and an abhorrently ugly syntax. Should've just went with Lua or Python, we'd all be better off.